src/coverage.ts

```diff
@@ -81,22 +99,36 @@ export const coverage = async ({
   })
 
   const { linesHit, linesFound } = diffCoverage
-  const { data: diffCoverageStatus } = await createCommitStatus({
-    token: utoken ? utoken : ghToken,
-    context,
-    commitContext: 'codelyze/patch',
-    state: 'success',
-    description:
-      linesFound > 0
-        ? `${percentString(linesHit / linesFound)} of diff hit`
-        : 'No diff detected'
-  })
+  const diffCoverageRate = linesHit / linesFound
+  console.log(
+    `diffCoverage: ${diffCoverageRate} | differenceThreshold: ${differenceThreshold}`
+  )
+  let diffCoverageStatus
+  if (!(emptyPatch && linesFound === 0)) {
+    const { data } = await createCommitStatus({
+      token: utoken ? utoken : ghToken,
+      context,
+      commitContext: 'codelyze/patch',
+      state: evaluateState([diffCoverageRate * 100 >= patchThreshold]),
```

Comment:
```suggestion
      state: evaluateState([diffCoverageRate * 100 >= patchThreshold]) ? 'success' : 'failure',
```

you could even extract a helper since you have two places where you need to do this.

```suggestion
      toState = success => success ? 'success' : 'failure'
      ...
      state: toState(evaluateState([diffCoverageRate * 100 >= patchThreshold])),
```

Reply:
applied